### DISCUSSION TITLE
config/navigation/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#188] [docs-comment]
# config/navigation/

概述 主题默认会自动生成最简单的导航栏配置，仅包括 首页 和 博客文章列表页 。 你也可以自己配置导航栏，覆盖默认的的导航栏配置。 默认配置如下： 当开启了 多语言配置，则会生成对应语言的导航栏的默认配置： 随着站点内容变得越来越丰富，包括了 博客文章列表、notes、友情链接、 外部链接等等， 默认生成的导航栏配置满足不了您的需求。 这时候，您可以通...

https://theme-plume.vuejs.press/config/navigation/

**Comment by bglove** [comment]
现在的导航栏，配置icon后。生成对应的网站结构时，svg图标外面有一个span标签，span标签有一个class=‘vp-icon’，他设置了background-color，导致icon有个黑色背景，很不好看。我自定义样式不知道为啥还覆盖不掉。

**Reply by pengzhanbo** [reply]
使用远程图标时，有 background-color 与 本地图标的属性冲突了，之前我在某一个版本中修复了，但可能是在另外的版本中出现了样式覆盖导致又重现了。我会在下个版本中修复这个问题。另外我更建议在 本地安装 `@iconify/json`，将图标本地化视觉体验会更好。

**Reply by bglove** [reply]
好的。我刚试了您说的本地安装图标库，重启之后就没有黑色背景。没想到佬这么快就回复了，Plume主题真的很好看，十分感谢您。

**Comment by xbhog** [comment]
问下主题有没有类似时间轴的组件可以放入导航栏中的。

**Reply by pengzhanbo** [reply]
我不大理解你的想法，请通过 issue 中的 feature request 详细描述下的诉求，主题再考虑是否作为新功能加入，或者给出其他的建议。

**Comment by liukunup** [comment]
请问下，为啥“笔记”这种路由是可以的，但是`/homelab/deploy/`这种路由前缀不行，已经创建了`docs/homelab/deploy/`目录，并在deploy下面写了一个test.md文档，预期是顶部导航栏点击后，像博客文章列表一样展示deploy下的文档

```javascript
export const zhNavbar = defineNavbarConfig([
  { text: '首页', link: '/', icon: 'material-symbols:home-outline' },
  { text: '博客', link: '/blog/' },
  { text: '标签', link: '/blog/tags/' },
  { text: '归档', link: '/blog/archives/' },
  {
    text: '笔记',
    items: [
      { text: 'Python', link: '/notes/python/搭建开发环境.md' },
      { text: 'Go', link: '/notes/go/搭建开发环境.md' },
    ]
  },
  {
    text: '家庭实验室',
    items: [
      { text: '架构设计', link: '/homelab/architecture/' },
      { text: '部署指南', link: '/homelab/deploy/' },
    ]
  },
])
```

**Reply by pengzhanbo** [reply]
更新到最新版本使用 collections 实现此功能吧。notes 已经被移除了。
